vapi/simulator/simulator.go

```diff
@@ -82,15 +83,16 @@ type download struct {
 
 type handler struct {
 	sync.Mutex
-	ServeMux    *http.ServeMux
-	URL         url.URL
-	Category    map[string]*tags.Category
-	Tag         map[string]*tags.Tag
-	Association map[string]map[internal.AssociatedObject]bool
-	Session     map[string]*rest.Session
-	Library     map[string]*content
-	Update      map[string]update
-	Download    map[string]download
+	ServeMux       *http.ServeMux
+	URL            url.URL
+	Category       map[string]*tags.Category
+	Tag            map[string]*tags.Tag
+	Association    map[string]map[internal.AssociatedObject]bool
+	Session        map[string]*rest.Session
+	Library        map[string]*content
+	Update         map[string]update
+	Download       map[string]download
+	ShutdownStatus shutdown.Status
```

Comment:
nit: should we rename the field to `Shutdown`?

Reply:
Renamed to `Shutdown`